.vscode/launch.json

```diff
@@ -9,12 +9,7 @@
             "program": "${workspaceRoot}/src/azure-cli/azure/cli/__main__.py",
             "cwd": "${workspaceRoot}",
             "args": [
-                "ts",
-                "export",
-                "-s",
-                 "/subscriptions/a1bfa635-f2bf-42f1-86b5-848c674fc321/resourceGroups/Gokul-TestRG4/providers/Microsoft.Resources/templateSpecs/testTS/versions/1",
-                "--output-folder",
-                 "C:\\Users\\daetienn\\Desktop\\ExportBugOutput2"
```

Comment:
This change was accidentally done by https://github.com/Azure/azure-cli/pull/18928.